projects/core/src/cart/store/effects/cart.effect.ts

```diff
@@ -72,7 +69,10 @@ export class CartEffects {
           switchMap((cart: Cart) => {
             if (cart.entries) {
               for (const entry of cart.entries) {
-                this.productImageConverter.convertProduct(entry.product);
+                entry.product = this.converter.convert(
```

Comment:
I'm not sure why we use `PRODUCT_NORMALIZER` in the cart effect. Maybe we can use CartNormalizer?

Reply:
Very good point, thanks!